governance/polls/Vault Compensation Poll 3

```diff
@@ -0,0 +1,54 @@
+
+title: Vault Compensation Poll
+summary: Signal your support for vault compensation
+discussion link: https://forum.makerdao.com/t/vault-compensation-plan-v2/3584
```

Comment:
You mentioned writing an explainer post in chat. This should link to that post (which should probably go up asap).

Reply:
https://forum.makerdao.com/t/vault-compensation-plan-v2/3584/97

Reply:
updated